base-theme/assets/webpack/webpack.prod.ts

```diff
@@ -20,7 +20,7 @@ module.exports = merge(common, {
       new TerserPlugin({
         parallel:      true,
         terserOptions: {
-          ecma: 6
+          ecma: 5
```

Comment:
When converting the config files to typescript, I discovered that `6` is not a valid valid for this configuration. The valid values are:
```
export type ECMA = 5 | 2015 | 2016 | 2017 | 2018 | 2019 | 2020;
```
This configuration option controls what sort of optimizations the terser plugin will do. The higher ES version you specify, the more optimizations it does, but the optimizations might be valid in fewer browsers. For example, the terser docs say that specifying 2015 can turn `{ x: x }` into `{ x }`. (I couldn't actually get this optimization to happen in their [playground](https://try.terser.org/), but whatever.)

Anyway, 6 is invalid. (The thing people call ES6 is formally ES2015).

5 is the default, which is what I suspect was being used before.